docs/eventing/broker/kafka-broker.md

```diff
@@ -0,0 +1,189 @@
+```
+
+The above `ConfigMap` is installed in the cluster, and you can edit
+the configurations or create a new one with the same shape 
+depending on your needs.
+
+## Set as default broker implementation
+
+To set the Kafka Broker as your default implementation you can edit
+the `config-br-defaults` `ConfigMap` in `knative-eventing` namespace.
+In this way, you can avoid specifying 
+`metadata.annotations.eventing.knative.dev/broker.class` or 
+`spec.config` for each Kafka Broker created in the cluster
+or in a specific namespace.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-defaults
+  namespace: knative-eventing
+data:
+  default-br-config: |
+    clusterDefault:
+      brokerClass: kafka 
+      apiVersion: v1
+      kind: ConfigMap
+      name: kafka-broker-config
+      namespace: knative-eventing
+    namespaceDefaults:
+      namespace1:
+        brokerClass: kafka 
+        apiVersion: v1
+        kind: ConfigMap
+        name: kafka-broker-config
+        namespace: knative-eventing
+      namespace2:
+        brokerClass: kafka 
+        apiVersion: v1
+        kind: ConfigMap
+        name: kafka-broker-config
+        namespace: knative-eventing
+```
+
+## Change Kafka Producer and Consumer configuration
+
+We expose all Kafka producer and consumer configurations, so that they can be 
+changed accordingly to specific workloads that it needs to run.
+
+Such configurations are in the `ConfigMap` `config-kafka-broker-data-plane`
+in the `knative-eventing` namespace. All configurations in that `ConfigMap`
+are documented on the [Apache Kafka website](https://kafka.apache.org/documentation/),
+in particular, [Producer configurations](https://kafka.apache.org/documentation/#producerconfigs)
+and [Consumer configurations](https://kafka.apache.org/documentation/#consumerconfigs).
+
+## Change logging level to data plane components
+
+The logging config for the data plane components looks like this:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kafka-broker-config-logging
+  namespace: knative-eventing
+data:
+  config.xml: |
+    <configuration>
+      <appender name="jsonConsoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="net.logstash.logback.encoder.LogstashEncoder"/>
+      </appender>
+      <root level="INFO">
+        <appender-ref ref="jsonConsoleAppender"/>
+      </root>
+    </configuration>
+```
+
+To change the log level to `DEBUG`, you need to apply the following `ConfigMap`:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kafka-broker-config-logging
+  namespace: knative-eventing
+data:
+  config.xml: |
+    <configuration>
+      <appender name="jsonConsoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="net.logstash.logback.encoder.LogstashEncoder"/>
+      </appender>
+      <root level="DEBUG">
+        <appender-ref ref="jsonConsoleAppender"/>
+      </root>
+    </configuration>
+```
```

Comment:
Do they need to create this from scratch or update an existing file?

Reply:
Update and existing `ConfigMap`, I clarified this point more.